connector/checkpoint/checkpoint_view.xml

```diff
@@ -48,6 +48,7 @@
                         delete="false" edit="false" version="7.0">
                     <field name="model_id"/>
                     <field name="record"/>
+                    <field name="message"/>
```

Comment:
Ah I see your point. Your latter idea of using `name` seems the best in terms of flexibility IMO.
 
The downside there would be an inability to sort on those columns, which I'm pretty sure some of my customers do. 

In your use case, how long are these messages? I looked at the default, and it seemed about 50 chars. Are the live messages about as long?